### PR TITLE
adds html only build

### PR DESCRIPTION
**Pull request by significance**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const posthtmlInlineAssets = require('posthtml-inline-assets');
+
+const inlineFonts = require('gulp-inline-fonts');
+
+exports.default = function() {
+  return src('dist/*.html')
+  .pipe(
+  posthtml([
+    posthtmlInlineAssets({
+    	cwd: '/Users/sig/Code/swarm2/swapchat/dist'
```
Comment (agazso) [inline gulpfile.js:13]: I think this should be `dist` instead

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  .pipe(inlineFonts({ name: 'DOS' }))
+  .pipe(dest('dist/'));
+};
+
+let inline_js_css = function() {
+  return src('dist/*.html')
+  .pipe(
+  posthtml([
+    posthtmlInlineAssets({
+    	cwd: '/Users/sig/Code/swarm2/swapchat/dist'
```
Comment (agazso) [inline gulpfile.js:30]: I think this should be `dist` instead

File: package.json [hunk 3]
```diff
@@ -31,6 +36,7 @@
     "@erebos/swarm": "^0.12.0",
     "@erebos/swarm-node": "^0.12.0",
     "eccrypto": "^1.1.3",
+    "gulp": "^4.0.2",
```
Comment (agazso) [inline package.json:39]: Shouldn't this be added to `devDependencies`?

**Comment by significance** [comment]
yes, good spots @agazso, thanks, i'll rectify these

**Pull request merged**